### DISCUSSION TITLE
REST Connector:  default headers between connectors do not seem to be unique

### DISCUSSION THREAD

**Discussion by ktalkington** [Budibase/budibase#4144] [Help]
Budibase 1.0.26

I created three REST API connectors, and need a default header "authorization" for each.  I add them fine but I noticed after inputting the first, the other two have the same header applied.  When editing the second, it updates the other two.  Should they not be unique?   Will open after an issue after I work out if this is end user issue or not.  Thanks.

**Comment by mjashanks** [comment]
Hi Karlan. This has been fixed in the new & improved REST API - I'm on `1.0.46`

**Reply by ktalkington** [reply]
updated to `1.0.46` and indeed it does have additional features, however, when I try to add an authentication method it seems the form validation of the dialog for entering auth is not working.  It detects when name field is empty but when entering information, the `add` button never enables

**Reply by mjashanks** [reply]
Hmm - it worked for me ok, tried Basic and Bearer - what browser are you using? 

![image](https://user-images.githubusercontent.com/3524181/150815689-b6a711a2-85d2-4b38-bc90-6a57a7b0dd35.png)
